internal/annotators/http/pki_test.go

```diff
@@ -54,19 +49,48 @@ func TestHttpPkiAnnotator_Do(t *testing.T) {
 		t.Fatalf(err.Error())
 	}
 
+	// Set up example signed data type for test purposes
+	type testData struct {
+		SignatureInput string
+		Signature      string
+	}
+
+	// Tests the Signature signed by the assembler called by buildRequest
+	t1 := testData{
+		SignatureInput: req.Header.Get("Signature-Input"),
+		Signature:      req.Header.Get("Signature"),
+	}
+
+	t2 := t1
+	t2.SignatureInput = "\"@method\" \"@path\" \"@authority\" \"Content-Type\" \"Content-Length\";created=1646146637;keyid=\"public.key\";alg=\"invalid\""
+
+	t3 := t1
+	t3.SignatureInput = "\"@method\" \"@path\" \"@authority\" \"Content-Type\" \"Content-Length\";created=1646146637;keyid=\"invalid\";alg=\"ed25519\""
+
+	t4 := t1
+	t4.Signature = ""
+
+	t5 := t1
+	t5.Signature = "invalid"
+
 	tests := []struct {
 		name        string
 		expectError bool
+		data        testData
 	}{
-		{"pki annotation OK", false},
-		{"pki bad key type", true},
-		{"pki key not found", true},
-		{"pki empty signature", false},
-		{"pki invalid signature", false},
+		{"pki annotation OK", false, t1},
+		{"pki bad key type", true, t2},
+		{"pki key not found", true, t3},
+		{"pki empty signature", false, t4},
+		{"pki invalid signature", false, t5},
 	}
 
 	for _, tt := range tests {
-		ctx := buildContext(tt.name, req)
+		req.Header.Set("Signature-Input", tt.data.SignatureInput)
+		req.Header.Set("Signature", tt.data.Signature)
+
+		ctx := context.WithValue(req.Context(), testRequest, req)
```

Comment:
w/r/t comment above RE: `testRequest` this action right here demonstrates what the client application consuming the SDK will need to do to make the request visible to the annotator, and thus why the constant needs to be exported and renamed.